data/rulei/rulei.txt

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2022 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Relic of the Past"
+	invisible
+	landing
+	to complete
+		never
+	npc save
+		government "Rulei"
+		personality heroic vindictive staying mute unconstrained frugal
+		system "Kanguwa"
+		ship "vyu-Ira" "Solemnity"
+
+mission "Ira Contact"
+	invisible
+	landing
+	to complete
+		never
+	on enter Kanguwa
+		fail
+		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago.`
+			`They communicate through some sort of invasive psionic ability.`
+		conversation
+			`Moments after arriving at Kanguwa, you are overwhelmed by an intense sense of emptiness. As you attempt to comprehend this feeling, you begin hearing whispers from behind.`
+			choice
+				`	(Turn around slowly.)`
+				`	(Pull out your sidearm.)`
+			`	You turn around, but nothing is there besides the familiar internals of your ship. The whispers remain behind your head, like an itch that can't be reached. The whispers are in a completely foreign language that you can't even begin to understand. As you keep looking around, you begin hearing more and more murmuring. There feels to be hundreds of distinct voices tugging your mind in every direction at once now, overloading your senses. Just as you think you can't take anymore, everything falls completely silent.`
+			choice
+				`	"Hello?"`
+				`	"Is there someone there?"`
+				`	"Show yourself!"`
+			scene "scene/rulei flash"
+			`	"Human."`
+			`	An image suddenly flashes in your head as you feel millions of eyes staring at you from beyond the boundaries of space. Looking around again, you see a strange-looking spacecraft a bit off into the distance in front of you facing your direction. The voices continue to surround your mind with only slightly reduced intensity.`
+			choice
+				`	"Who are you?"`
+				`	"Where am I?"`
+					goto where
+			`	Numerous voices start up, but rather than becoming louder, they go back and forth in your head, as though they are having a discussion. After a few seconds, they fall silent, then speak a single word.`
```

Comment:
```suggestion
			`	Numerous voices start up, but rather than becoming louder, they go back and forth in your head, as though they are having a discussion. After a few seconds, they fall silent, then speak a single word in a cacophonous unison.`
```